keyboards/flehrad/tradestation/rules.mk

```diff
@@ -2,17 +2,10 @@
 MCU = atmega32u4
 
 # Bootloader selection
-#   Teensy       halfkay
-#   Pro Micro    caterina
-#   Atmel DFU    atmel-dfu
-#   LUFA DFU     lufa-dfu
-#   QMK DFU      qmk-dfu
-#   ATmega32A    bootloadHID
-#   ATmega328P   USBasp
 BOOTLOADER = caterina
 
 # Build Options
-#   comment out to disable the options.
+#   change yes to no to disable
 #
 BOOTMAGIC_ENABLE = full	# Virtual DIP switch configuration(+1000)
```

Comment:
```suggestion
BOOTMAGIC_ENABLE = full	# Virtual DIP switch configuration
```

Reply:
This is line will be updated on the `develop` branch when #13846 is merged in.